_docs/operations/updates.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: guide
+title: StorageOS Docs - Update StorageOS version
+anchor: operations
+module: operations/updates
+---
+
+# Update StorageOS version
+
+> Before upgrading, check the 
+> [release notes]({%link _docs/reference/release_notes.md %}) to confirm
+> whether there is a safe upgrade path between versions.
+
+
+StorageOS version upgrades must be planned and executed taking in consideration
+that data will remain inaccessible during the process. It is recommended to
+schedule a maintenance window.
+
+There are two strategies to update StorageOS.
```

Comment:
There are currently two strategies to upgrade StorageOS. (we should imply, or explicitly state, that these options will get better with time)

Reply:
I added this:

There are two strategies to upgrade StorageOS for now. And both keep data
consistency during the upgrade process. StorageOS keeps the data in the host
where it is running. When the new version of StorageOS starts, the volumes from
the previous version are available.

1. Full stop of the cluster
1. Rolling update

> More upgrade procedures will be released that will automate the main part of
> the process and fulfil use cases not covered currently.

Reply:
done